counter.js

```diff
@@ -0,0 +1,37 @@
+// Counter component
+const counter = (function (){
+  let count = {
```

Comment:
You're resetting default values in `setCounter` method. So, I suppose that you don't need them here. Just put `const count = {}` here.